src/main/java/racingcar/IOclass.java

```diff
@@ -0,0 +1,56 @@
+package racingcar;
+import java.util.Scanner;
+
+public class IOclass {
+    public static String[] getName(){
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        Scanner sc = new Scanner(System.in);
+        String str=sc.nextLine();
+        String[] cars = str.split(",");
+        return cars;
+    }
+    
+    public static boolean checkCarName(String[] carName){
+        for(int i=0; i< carName.length; i++){
+            if(carName[i].length()>5)
+                return false;
+        }
+        return true;
+    } 
+    public static int getTimes(){
```

Comment:
```suggestion
    public static boolean checkCarName(String[] carName){
        for(int i=0; i< carName.length; i++){
            if(carName[i].length()>5)
                return false;
        }
        return true;
    } 
    
    public static int getTimes(){
```
[구글 코드 컨벤션 - vertical-whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace)에 따르면 `메서드와 메서드 사이`에는 한 줄 띄어쓰라고 가이드하고 있어요! 

현재 `IOClass`의 코드를 살펴봤을 때, IOClass의 역할은 아래와 같습니다. 
* `자동차의 이름을 입력받는다`,
* `자동차 이름의 유효성을 검증한다`
* `시도 횟수를 입력받는다` 
* `자동차의 최장 거리(?)를 찾는다`
*`우승자를 출력한다`(+ `우승자를 찾는다`)

하나의 객체가 너무 많은 역할을 하고 있는 것으로 보입니다. 
IOClass가 가지고 있는 여러 책임들을 다른 객체에 적절히 위임하여 객체의 목적을 좀 더 명학확하게 하는 것은 어떨까요? 🤔 
(Tip: 먼저 `Car` 객체를 만들어보시면 어떨까요? )

[SOLID 1편 SRP와 OCP](https://tecoble.techcourse.co.kr/post/2020-07-31-solid-1/) (해당 글에서 SRP의 내용만 봐주세요! 예제는 잘 모르는 개념이실 수 있으니 가볍게 봐주세요 👀)

Reply:
말씀해주신대로 Car 객체를 따로 만들었습니다.
Car 객체에 속성으로 이름과 위치를 줬는데, IOClass에 있는 어떤 메소드를 Car로 분담시켜야 될지 잘 모르겠어서 질문드립니다.

Reply:
먼저 IOClass가 어떤 역할을 하는지를 먼저 정하시는게 좋을 것 같아요! 😃
IOClass는 `입력과 출력을 담당하는 역할`이라고 생각되네요. 🤔  (이 둘도 분리하면 좋을 것 같아요 🧐) 
`자동차의 이름을 입력받는다,`, `시도 횟수를 입력받는다`, `우승자를 출력한다`(여기서는 언급하지 않았지만, 자동차의 위치를 출력하거나 입력하라는 메시지 출력 또한...)는 IOClass의 역할이 될 것 같습니다.  

이제 남은 역할들을 살펴보면 아래와 같아요! 👀
* 자동차의 이름의 유효성을 검증한다. 
* 자동차의 최장 거리를 찾는다. 
* 우승자를 찾는다.  

이 중에서 `자동차 이름의 유효성을 검증`해야할 책임이 있지 않을까요? 🤔